DM-42606: Update to reflect new schema registry configurations (#8)
lsst-dm/dmtn-093: index.rst

In some cases, a break with compatibility may be required (for example, when some particular data product is rendered obsolete, deleting the corresponding field from the alert schema will break the ``FORWARD_TRANSITIVE`` compatibility guarantee).
Such a break with compatibility will be signified by the release of a new ``MAJOR`` version of the schema.
Issuing a new major schema will require that consumers not using the schema registry to manually update their kafka
deserializer to use the newest schema.

The Confluent Schema Registry makes it possible for schemas to evolve within a given *subject name*. New schemas
added to a given subject name will have a unique id and version number. To maintain a consistent registry history,
the ``FORWARD_TRANSITIVE`` compatibility is not enforced within the schema. All major and minor

> schema changes are instead given a unique schema id based on the major and minor version. For example, schema version

Reviewer: ```suggestion
schema changes are instead given a unique schema id based on the major and minor version defined in `lsst/alert_packet`_. For example, schema version
```

> 7.1 is given schema id 701. For schema version 13.12, we would have schema id 1312. This allows us to maintain a

Reviewer: ```suggestion
7.1 is given schema id 701. For schema version 13.12, we would have schema id 1312. This allows us to maintain
```

> consistency so all alerts can be read using the schema registry.

Reviewer: ```suggestion
consistency so all historical alerts can be read even if the schema registry has to be recreated.
```
